test/version_test.cpp.in

```diff
@@ -0,0 +1,27 @@
+#include "yaml-cpp/version.h"
+
+#include "gtest/gtest.h"
+
+namespace YAML {
+namespace {
+TEST(VersionTest, Major)
```

Comment:
Put { on the same line as the function.

(For that matter, make sure you run it through clang-format using the style file at the base of the repo.)